pages/index.tsx

```diff
@@ -356,14 +378,21 @@ export default function Home({ featuredStorefronts }: HomeProps) {
     sortWith(sorts[sortBy])
   );
 
+    
+
   // initial fetch and display
   useEffect(() => {
     async function getListings() {
       const allListings = await IndexerSDK.getListings();
+      let daoFilteredListings = allListings;
+
+      if (WHICHDAO) {
+        daoFilteredListings = daoFilteredListings.filter(listing => selectedDaoSubdomains.includes(listing.subdomain))
+      }
```

Comment:
Recommend passing in selected subdomains into the getListings call. selectedSubdomains defaults to undefined which excludes the filter from running within getListings.

```suggestion
      const allListings = await IndexerSDK.getListings({ subdomains: selectedSubdomains });
```